runner/utils/driver/docker/docker.go

```diff
@@ -376,6 +379,25 @@ func getServiceContainerName(projectID, serviceID, version, taskID, clusterID st
 	return fmt.Sprintf("space-cloud-%s-%s--%s--%s--%d--%s", clusterID, projectID, serviceID, version, index, taskID)
 }
 
+func splitServiceContainerName(containerName string) (clusterID, projectID, serviceID, version, taskID string) {
+	s := strings.Split(containerName, "--")
+	spaceCloudPrefixStr := strings.Split(s[0], "-")
+	if len(spaceCloudPrefixStr) == 3 {
+		// default cluster
+		return "default", spaceCloudPrefixStr[1], s[1], s[2], s[4]
+	}
+	return spaceCloudPrefixStr[1], spaceCloudPrefixStr[1], s[1], s[2], s[4]
```

Comment:
Are you sure the right things are being returned here? Can you comment the possible formats?

Reply:
if you check the helper functions written above this function, you will get the idea of possible formats

Reply:
```suggestion
	return spaceCloudPrefixStr[2], spaceCloudPrefixStr[3], s[1], s[2], s[4]
```